### PR TITLE
[Merged by Bors] - ET-4278 Changing scripts to download data from AWS S3

### PR DESCRIPTION
**Pull request by jorge-cr-13**

### CODE REVIEW

File: .github/scripts/download_assets.sh [hunk 1]
```diff
@@ -22,13 +22,12 @@ download()
...
   TMP_DIR=$(mktemp -d)
-  URL="http://s3-de-central.profitbricks.com/xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
+  URL="s3://xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
 
   if [ -d "$DATA_DIR/$ARCHIVE_BASENAME" ]; then
     echo "skip downloading $DATA_DIR/$ARCHIVE_NAME"
   else
-
-    curl "$URL" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
+    aws s3 cp $URL $DATA_DIR/$TMP_ARCHIVE_NAME
```
Comment (acrrd) [inline .github/scripts/download_assets.sh:30]: :question: I guess this is needed because now the bucket is private

Comment (acrrd) [reply .github/scripts/download_assets.sh]: Ok then we need to make sure everyone in the team and the CI can access the new one before we merge this.

File: .github/scripts/download_assets.sh [hunk 2]
```diff
@@ -39,6 +39,7 @@ download()
     shasum -c "$CHECKSUM_FILE"
 
     mv "$TMP_DIR/$ARCHIVE_BASENAME" "$DATA_DIR/"
+    cd "$DATA_DIR"
```
Comment (acrrd) [inline .github/scripts/download_assets.sh:42]: Can you move this at the beginning of the function instead?

File: .github/scripts/download_assets.sh [hunk 3]
```diff
@@ -22,13 +22,15 @@ download()
...
   TMP_DIR=$(mktemp -d)
-  URL="http://s3-de-central.profitbricks.com/xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
+  URL="s3://xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
 
   if [ -d "$DATA_DIR/$ARCHIVE_BASENAME" ]; then
     echo "skip downloading $DATA_DIR/$ARCHIVE_NAME"
   else
 
-    curl "$URL" -o "$DATA_DIR/$TMP_ARCHIVE_NAME" -C -
+    cd "$DATA_DIR"
```
Comment (acrrd) [inline .github/scripts/download_assets.sh:31]: since the `if` above uses `$DATA_DIR` we can move this before it.

**Review by acrrd — APPROVED** [review]

**Comment by jorge-cr-13** [comment]
bors merge

**Pull request closed without merging**